app/Media/MediaServiceProvider.php

```diff
@@ -150,10 +150,11 @@ function () use ($container) {
         );
 
         $container->addShared(
-            ImagesToMediaObjectReferencesConvertor::class,
+            ImageNormalizer::class,
```

Comment:
If we expose `ImageNormalizer` here I would also change the other usages, or we also inline the constructor in the service providers. Now we have a mix.

Reply:
Modified with https://github.com/cultuurnet/udb3-backend/pull/1960/commits/cb7543479b4a848bfb92132fb8f8c97bf9a28962